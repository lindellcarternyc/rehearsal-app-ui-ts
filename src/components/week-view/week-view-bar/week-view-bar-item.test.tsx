import * as React from 'react'
import * as renderer from 'react-test-renderer'

import WeekViewBarItem, { WeekViewBarItemProps } from './week-view-bar-item'

const onClick = jest.fn()

describe('WeekViewBarItem', () => {
  it('renders correctly without rehearsals', () => {
    const withoutRehearsals: WeekViewBarItemProps = {
      dayName: 'S',
      dayNumber: '14',
      hasRehearsals: false,
      onClick
    }
    const tree = renderer.create(
      <WeekViewBarItem {...withoutRehearsals} />
    )
    expect(tree).toMatchSnapshot()
  })

  it('renders correctly with rehearsals', () => {
    const withRehearsals: WeekViewBarItemProps = {
      dayName: 'T',
      dayNumber: '16',
      hasRehearsals: true,
      onClick
    }
    const tree = renderer.create(
      <WeekViewBarItem {...withRehearsals} />
    )
    expect(tree).toMatchSnapshot()
  })
})