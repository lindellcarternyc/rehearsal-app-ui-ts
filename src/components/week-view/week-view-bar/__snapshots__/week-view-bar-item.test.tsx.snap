// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekViewBarItem renders correctly with rehearsals 1`] = `
<div
  className="item"
  onClick={[Function]}
>
  <h3
    align="center"
    className="ui header"
  >
    <div
      className="sub header"
    >
      T
    </div>
    16
  </h3>
  <div>
    <i
      aria-hidden="true"
      className="circle tiny icon"
    />
  </div>
</div>
`;

exports[`WeekViewBarItem renders correctly without rehearsals 1`] = `
<div
  className="item"
  onClick={[Function]}
>
  <h3
    align="center"
    className="ui header"
  >
    <div
      className="sub header"
    >
      S
    </div>
    14
  </h3>
</div>
`;
